src/app/armory/AllWishlistRolls.tsx

```diff
@@ -16,7 +16,15 @@ import styles from './AllWishlistRolls.m.scss';
  * (with some overrides to set some as "plugged", when spawned from a real item).
  * This would render much weirder if it were fed an owned inventory item.
  */
-export default function AllWishlistRolls({ item }: { item: DimItem }) {
+export default function AllWishlistRolls({
+  item,
+  realAvailablePlugHashes,
+}: {
+  item: DimItem;
+  // non-plugged, but available, plugs, from the real item this was spawned from.
+  // used to mark sockets as available
```

Comment:
My preference is to use `/** */` jsdoc comment blocks for things you expect to show up in VSCode intellisense